R.swift/main.swift

```diff
@@ -13,13 +13,28 @@ let IndentationString = "  "
 
 let ResourceFilename = "R.generated.swift"
 
+private func isValidURL(ignoreFile : IgnoreFile?, url: NSURL) -> Bool {
```

Comment:
I think it would be simpler to create a `IgnoreFile.init()` without any arguments that creates an empty ignore file. Then all the logic can stay in one place and there are less exception.